### DISCUSSION TITLE
Blank mp3 files

### DISCUSSION THREAD

**Discussion by Wildheart-Baby** [shaked6540/YoutubePlaylistDownloader#91] [General]
I downloaded the latest version of the app and have noticed that some of my mp3 files are silent when downloaded as part of a playlist, but redownloading as a single mp3 and they are fine.

**Comment by shaked6540** [comment]
Can you give me a link to a playlist that causes this, and also the settings you used?

**Reply by Wildheart-Baby** [reply]
I think this one had an issue with let it bleed
https://www.youtube.com/playlist?list=PLiN-7mukU_RGbkLwK8g0SIBCAwLlgKwnb

I had the default settings, I'd never changed them

**Reply by shaked6540** [reply]
I tried reproducing the problem but I have audio on the downloaded files, are you sure you're using the default settings?
